pkg/config/codegen/parse.go

```diff
@@ -0,0 +1,251 @@
+package main
```

Comment:
This file is the main diff (it was renamed from `main/parse.go`, but unfortunately GitHub is not smart enough to tell)